sidebars.js

```diff
@@ -583,23 +583,63 @@ const sidebars = {
           items: [
             {
               type: "doc",
-              label: "General",
-              id: "developer-docs/security/general-security-best-practices",
+              label: "Overview",
+              id: "developer-docs/security/security-best-practices/overview",
+            },
+            {
+              type: "doc",
+              label: "Inter-canister calls",
+              id: "developer-docs/security/security-best-practices/inter-canister-calls",
+            },
+            {
+              type: "doc",
+              label: "Identity and access management",
+              id: "developer-docs/security/security-best-practices/iam",
+            },
+            {
+              type: "doc",
+              label: "Decentralization",
+              id: "developer-docs/security/security-best-practices/decentralization",
+            },
+            {
+              type: "doc",
+              label: "Data integrity and authenticity",
+              id: "developer-docs/security/security-best-practices/data-integrity-and-authenticity",
+            },
+            {
+              type: "doc",
+              label: "Data storage",
+              id: "developer-docs/security/security-best-practices/data-storage",
+            },
+            {
+              type: "doc",
+              label: "HTTP outcalls",
+              id: "developer-docs/security/security-best-practices/https-outcalls",
+            },
+            {
+              type: "doc",
+              label: "Denial of Service",
+              id: "developer-docs/security/security-best-practices/dos",
+            },
+            {
+              type: "doc",
+              label: "Canister upgrades",
+              id: "developer-docs/security/security-best-practices/canister-upgrades",
```

Comment:
@jessiemongeon1 I see you are ordering them alphabetically, however, I ordered them so that the most important aspects are on top. Would it be OK to keep that ordering?

Reply:
Discussed on Slack: It is ok with Jessie to keep the ordering as that was intentional. Resolving this.